app/src/main/java/com/example/android/navigation/GameOverFragment.kt

```diff
@@ -31,7 +31,7 @@ class GameOverFragment : Fragment() {
         val binding: FragmentGameOverBinding = DataBindingUtil.inflate(
                 inflater, R.layout.fragment_game_over, container, false)
         binding.tryAgainButton.setOnClickListener() {
```

Comment:
If not pressing on the try again button, but pressing on the back button the behaviour is not the same but it should be:
Press try again -> goes back to question 1
Press back button -> goes back to home screen

Reply:
Yes, the back button behaviour is changed because I added popUpTo Title fragment behaviour in the action.
(back stack manipulation)